rest-request.go

```diff
@@ -37,10 +38,29 @@ var DefaultHTTPClient = http.DefaultClient
 
 // Client uses Grafana REST API for interacting with Grafana server.
 type Client struct {
-	baseURL   string
-	key       string
-	basicAuth bool
-	client    *http.Client
+	baseURL       string
+	key           string
+	basicAuth     bool
+	client        *http.Client
+	customHeaders map[string]string
+}
+
+// SetCustomHeaders - set additional headers that will be sent with each request
+func (r *Client) SetCustomHeaders(headers map[string]string) {
+	r.customHeaders = headers
+}
+
+// SetCustomHeader - set additional header that will be sent with each request
+func (r *Client) SetCustomHeader(key, value string) {
+	if r.customHeaders == nil {
+		r.customHeaders = map[string]string{}
+	}
+	r.customHeaders[key] = value
+}
+
+// SetOrgIDHeader - set `X-Grafana-Org-Id`  to specified value
+func (r *Client) SetOrgIDHeader(oid uint) {
```

Comment:
I'm not sure if we should have this. It was undocumented for a long time and we have https://github.com/grafana-tools/sdk/blob/master/rest-org.go#L145-L146 for this purpose. IMO if someone wants this then they should really set it manually themselves.

Reply:
>https://github.com/grafana-tools/sdk/blob/master/rest-org.go#L145-L146

this one actually differ, cause if you are an admin you need to switch context to use it, but it won't work in parallel at all